FluentStorage.SFTP/ConnectionFactory.cs

```diff
@@ -24,7 +24,7 @@ class ConnectionFactory : IConnectionFactory {
 				connectionString.GetRequired("host", true, out string host);
 				connectionString.GetRequired("user", true, out string user);
 				connectionString.GetRequired("password", true, out string password);
-				var path = connectionString.Get("path") ?? "";
```

Comment:
how is this a fix?

Reply:
It's a fix because this variable (null by default) is copied to the SshNetSftpBlobStorage.RootDirectory field. Then `StoragePath.Combine(RootDirectory... ` which is used to map to the correct directory, works faster if one of the params is a null (see source)
```cs
		public static string Combine(IEnumerable<string> parts) {
			if (parts == null) return Normalize(null);

			return Normalize(string.Join(PathSeparatorString, parts.Where(p => p != null).Select(p => NormalizePart(p))));
		}
```

Before instead of null it was an empty string, which was ok, but null makes more sense.